front/src/actions/message.js

```diff
@@ -93,15 +110,12 @@ function createActions(store) {
         });
         newState.messages = sortMessages(newState.messages);
         store.setState(newState);
-        actions.scrollToBottom();
         // then we send the message
-        const createdMessage = await state.httpClient.post('/api/v1/message', newMessage);
-        const messagesWithoutTempMessage = store.getState().messages.filter(message => message.tempId !== tempId);
-        messagesWithoutTempMessage.push(createdMessage);
+        await state.httpClient.post('/api/v1/message', newMessage);
         // then we remove the message loading
         const finalState = update(state, {
           messages: {
-            $set: sortMessages(messagesWithoutTempMessage)
```

Comment:
why did you remove the sort? 

The goal of the sort is to avoid the case when you ask something to Gladys, and the replies appears before the question.

Reply:
Fixed